datamodel/app/triggers/create_triggers.sql

```diff
@@ -0,0 +1,73 @@
+-- only update -> insert and delete are handled by reach trigger
```

Comment:
I'm not a fan of separating the code which create the trigger from the ones which create the trigger functions.

Reply:
I've brought back the triggers but organized respectively the functions with their triggers.